### PR TITLE
Feature/49294

### PR DESCRIPTION
**Pull request by bartdominiak**
https://dev.snowdog.pro/issues/49294

### CODE REVIEW

File: Magento_Catalog/templates/product/image_with_borders.phtml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <div class="lazyload-wrapper">
     <img
-        class="image lazyload product-image-photo"
+        class="image catalog-grid-item__image lazyload"
```
Comment (Aekal) [inline Magento_Catalog/templates/product/image_with_borders.phtml]: Please dont remove `product-image-photo`, it's used in `swatch-renderer` script

Comment (szafran89) [reply Magento_Catalog/templates/product/image_with_borders.phtml]: It would be great to add the comment for what is this class

Comment (bartdominiak) [reply Magento_Catalog/templates/product/image_with_borders.phtml]: Ok. Didn't know that. I will leave it with `.product-image-photo`

**Comment by Aekal** [comment]
#122

File: Magento_Downloadable/templates/catalog/product/type.phtml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        <div
+            class="product-view__stock-status"
+            title="<?= __('Availability') ?>"
+        >
+            <?= __('In stock') ?>
+        </div>
+    <?php else: ?>
+        <div
+            class="product-view__stock-status product-view__stock-status--alert"
+            title="<?= __('Availability') ?>"
```
Comment (Aekal) [inline Magento_Downloadable/templates/catalog/product/type.phtml]: Missing closing `>`

File: Magento_Downloadable/templates/catalog/product/type.phtml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            <?= __('Out of stock') ?>
+        </div>
+    <?php endif; ?>
+<?php endif; ?>
+
+<?= $block->getChildHtml() ?>
+
```
Comment (Aekal) [inline Magento_Downloadable/templates/catalog/product/type.phtml]: Remove empty line

File: Magento_Downloadable/templates/catalog/product/type.phtml (hunk 2 above)
Comment (bartdominiak) [reply Magento_Downloadable/templates/catalog/product/type.phtml]: Done

File: Magento_Downloadable/templates/catalog/product/type.phtml (hunk 3 above)
Comment (bartdominiak) [reply Magento_Downloadable/templates/catalog/product/type.phtml]: Done

File: Magento_Downloadable/templates/catalog/product/type.phtml (hunk 2 above)
Comment (Aekal) [reply Magento_Downloadable/templates/catalog/product/type.phtml]: Wrong place, text should be inside div ;)

Comment (bartdominiak) [reply Magento_Downloadable/templates/catalog/product/type.phtml]: My fault 👍

**Pull request merged**